atd-toolbox/read_replica_proxy/haproxy.cfg.template

```diff
@@ -5,8 +5,8 @@ defaults
   log global
   mode tcp
   timeout connect 5000ms
-  timeout client 50000ms
-  timeout server 50000ms
+  timeout client 600000ms
```

Comment:
i think that these can be [represented in minutes](https://www.haproxy.com/documentation/haproxy-configuration-manual/latest/#2.5) if we wanted. Not really a big deal - i was just curious about it myself.

Reply:
Such a good call! Thank you - this is much more readable. https://github.com/cityofaustin/atd-vz-data/pull/1397/commits/a801f9a1b09dcbebcae3a32c858ce351a71c0f89